src/lib/colourType.js

```diff
@@ -12,4 +13,4 @@ export const isHex = colour => {
 
 export const isRGBorHSL = colour => /^(rgb|hsl)a?\(.*?\)/.test(colour);
 
-export const isKeyword = colour => has(colourNames, colour.toLowerCase());
+export const isKeyword = colour => has.call(colourNames, colour.toLowerCase());
```

Comment:
Well in that case, why doesn't you `Object.freeze` `colourNames`, yank `toLowerCase` from the `String.prototype`, require some extra package to use `call` everywhere (since it's notoriously hard to yank and then use) and hope that you didn't forget something.

If you mean that there are more then two packages that modify builtin prototypes, then yes you are most likely right. But if you just have a feeling there might be many, not including polyfills, that is not good enough.

I wonder if joyent could answer this question, because I don't know of a way to search in the source code of every packages on npm, let alone run them.